docs/releasing.md

```diff
@@ -140,7 +140,7 @@ npm publish
 # Publish viewer.
 yarn deploy-viewer
 
-# Publish the extension (if it changed).
+# Publish the extensions (if it changed).
```

Comment:
Are there firefox specific instructions so that someone can release this?  Or will that be handled in a follow up once this has landed and can be released for the first time?

Reply:
I haven't uploaded to the FF store before, so IDK. I'll update when I do.